book/peer_review.md

```diff
@@ -92,37 +115,51 @@ Be critical and clear, but not mean. Ask questions and set actions.
+
+```{important}
+If a project is only reviewed when all of the code has been written, this significantly reduces the benefit of review.
+
+This creates a much larger burden on the reviewer. Additionaly, any issues that are identified may take a lot of time to fix across the whole project. A reviewer might highlight that certain quality assurance practices have not been used - for example, there has not been enough documentation or automated testing in the project. It would take a substantial amount of effort to add documentation and testing for the whole project. If this was identified earlier, the improved practices could be applied as the remaining code is developed.
+```
 
-This form of review works well when changes to the code are small and frequent. Requesting review of small but regular changes reduces the burden on reviewers, relative to large review of a complete project. When carrying out a review of larger pieces of work, it may be worth reviewing different aspects of the code in separate sessions. For example, focussing on documentation in one session and functionality in the next.
+When you must carry out a review of larger or complete pieces of work, it may be worth reviewing different aspects of the code in separate sessions. For example, focussing on documentation in one session and functionality in the next.
 
 The thought of someone else reviewing your code in this way encourages good practices from the outset:
 * Clear code and documentation - so that others with no experience can use and test your code
 * Usable dependency management - so that others can run your code in their own environment
 
-Remote review is aided by features of most version control platforms, namely the pull request (or equivalent). See [](version_control.md) for more information.
+Separate review is aided by features of most version control platforms. See [](version_control.md) for more information.
 
-#### Case studies
+#### Case study - rOpenSci review
 
-Here we discuss an example from [rOpenSci](https://ropensci.org/); a community led initiative that curates open source, statistical R packages. rOpenSci apply a rigorous peer review process to assure the quality of packages before including them in their collection. This peer review process is entirely remote and is performed in the open, via GitHub pull requests.
+Here we discuss an example from [rOpenSci](https://ropensci.org/); a community led initiative that curates open source, statistical R packages. rOpenSci apply a rigorous peer review process to assure the quality of packages before including them in their collection. This peer review process is separate and is performed in the open, via GitHub pull requests.
```

Comment:
FYI I'd never known where to find the link for this case study until looking at the raw markdown, but this might be a me problem 😅